### DISCUSSION TITLE
How to set default timezone in my app please ?

### DISCUSSION THREAD

**Discussion by ningji** [js-fns/date-fns#3591] [Q&A] [answered]
Hi i did a quick search but cannot find the answer yet, in moment you can do
moment.tz.setDefault("America/New_York");
this sets the default timezeone.

What's the equivalent in date-fns please ?

**Comment by 25747** [comment, marked as answer]
There is not an equivalent with date-fns. You can achieve the same outcomes by making your own wrappers around `date-fns-tz` functions, but it is not like using moment.

**Reply by ningji** [reply]
Thx very much for the info !

**Comment by ningji** [comment]
Have a followup question, say i set the TZ to be New York timezone in the .env, and node.js app imports dotenv package to use New York time zone.
Now my docker container is using UK timezone as system time.

I know date-fns.format() generates output string in local timezone format, so in my case will it format into UK time or NY time ? @25747

**Reply by ningji** [reply]
ok i did quick test, seems it's using the timezone from my .env.

However, my unit test using jest still picks up in my local time in PC, not from .env :(
